lab-brianna/test/test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const assert = require('assert');
+const parcel = require('../lib/call-apply');
+
+describe('parcel', function() {
```

Comment:
**TODO FOR LOST POINTS**

I know you're working on your other tests, but good job so far for the tests you have! Remember, per the instructions, you'll need **2 tests** for ``call`` and ``apply.``